packages/puppeteer-core/src/common/Input.ts

```diff
@@ -682,4 +682,38 @@ export class Touchscreen {
       modifiers: this.#keyboard._modifiers,
     });
   }
+
+  async press(x: number, y: number, delay: number) {
```

Comment:
please add tests!

Reply:
thanks for reply, you mean unit test? 
I only have little unit test experiment on solidity.
you can say I have completely no knowledge about how to start
(for me, the whole folder looks like combine different type of coding language)

can you please give me some hint?(maybe about what language I should search,
if I want to add a test what folder I should focus, or documentation)

Reply:
See https://github.com/puppeteer/puppeteer/blob/main/test/src/touchscreen.spec.ts#L25 as an example and https://pptr.dev/contributing to learn how to run tests. The tests should verify that the CDP methods called here actually have the desired effect on the web page.

Reply:
thanks for reply, I have some problem about run tests,I have record the video, can you help me take a look at it? 

video:https://www.youtube.com/watch?v=CFZoN3mahVY
platform: macbook m1 pro

Reply:
npm install seems to failing on your machine (cannot see the full log on the video). You don't need to install Firefox, you can start testing with Chromium that is downloaded by default.

Reply:
here is the full log:https://drive.google.com/file/d/1WokvbV-dlZmQljjGdNCSefmMDOteCLr3/view?usp=share_link

pls take a look <(_ _)>

Reply:
when I use yarn, it fail with 
Couldn't find package "@puppeteer-test/installation@latest"

Reply:
what's your node and npm versions? yarn might not work as we don't use it for development.

Reply:
ah yeah, you are right, I should change to the highest version
here is my version, I will reinstall node and npm and try to npm install puppeteer again

my node version:
v18.12.1
npm version:
6.14.13

Reply:
Updating npm might help! I am using node v18.12.1 and npm 8.19.2, also on a M1 macbook

Reply:
one last question, can I test specific script? like npm run test touchscreen.spec

Reply:
put `it.only` instead of `it` for the tests you want to run, rebuild and run the test command. It will only run the specified tests.

Reply:
> 

hello, I add the test case, pls check <(_ _)>,if you have any problem, pls leave a comment

Reply:
will take a look shortly. I have started the bots, let's make sure bots are passing in the meantime.

Reply:
I upload the documentation and prettier